.test-infra/jenkins/job_PerformanceTests_CdapIO.groovy

```diff
@@ -24,7 +24,7 @@ String jobName = "beam_PerformanceTests_Cdap"
 
 job(jobName) {
   common.setTopLevelMainJobProperties(delegate)
-  common.setAutoJob(delegate, 'H H/6 * * *')
+  common.setAutoJob(delegate, 'H H/12 * * *')
```

Comment:
Actually, I think we can run perf tests even only once per day.

Reply:
yeah agree. When more performance tests are added we can further reduce the frequency. Just a trivial change.